File: December - 15/KishoreMuruganantham_royal_family_seating.swift
import Foundation
if let line = readLine() {
    var l = line
    if let r = l.range(of: "=") {
        l = String(l[r.upperBound...])
    }
    l = l.replacingOccurrences(of: "[", with: "")
    l = l.replacingOccurrences(of: "]", with: "")
    l = l.replacingOccurrences(of: " ", with: "")
    
    let parts = l.split(separator: ",")
    var valid = true
    var seenNull = false
    
    for p in parts {
        let s = String(p)
        if s == "null" {
            seenNull = true
        } else {
            if seenNull {
                valid = false
                break
            }
        }
    }
    print(valid ? "true" : "false")
}

File: December - 15/KishoreMuruganantham_royal_family_seating.R
input <- file("stdin", "r")
line <- readLines(input, n = 1)
line <- sub(".*=\\s*", "", line)
line <- gsub("\\[|\\]", "", line)
line <- gsub("\\s+", "", line)
parts <- unlist(strsplit(line, ","))
vals <- ifelse(parts == "null", NA, as.integer(parts))

valid <- TRUE
seen_null <- FALSE
for (v in vals) {
  if (is.na(v)) {
    seen_null <- TRUE
  } else {
    if (seen_null) {
      valid <- FALSE
      break
    }
  }
}
cat(tolower(as.character(valid)), "\n", sep = "")
